rb/spec/unit/selenium/webdriver/chrome/service_spec.rb

```diff
@@ -49,6 +49,41 @@ module WebDriver
           expect(service.uri.to_s).to eq "http://#{Platform.localhost}:#{port}"
         end
 
+        it 'allows #driver_path= with String value' do
+          path = '/path/to/driver'
+          Chrome::Service.driver_path = path
+
+          service = Service.chrome
+
+          expect(service.executable_path).to eq path
+        end
+
+        it 'allows #driver_path= with Proc value' do
+          path = '/path/to/driver'
+          proc = Proc.new { path }
```

Comment:
This might fall afoul of some of the later ruby cops. If you use `a_proc = proc { path }` it should remedy it. (For others too)